gtsfm/loader/argoverse_dataset_loader.py

```diff
@@ -0,0 +1,132 @@
+""" Loader that reads images from Argoverse directories on disk.
```

Comment:
Whats the convention for white space at the start of the docstring. I use a plugin in VS-Code to generate the template for me, and it doesn't leave that space. 

Just want to check if leaving a space is a good practice?

Reply:
not sure -- let's go with no space after triple-quote i.e. `"""This is a comment ...` for now